File: backend/Bioskopina.Model/Bioskopina.cs
using Bioskopina.Model;

namespace Bioskopina.Model
{
    public class Bioskopina
    {
        public int Id { get; set; }

   
        public string? TitleEn { get; set; }
       
        public string? Synopsis { get; set; }

        public string? TrailerUrl { get; set; }
        public string? ImageUrl { get; set; }

        public string? Director { get; set; }

        public int? YearRelease { get; set; }

        public int? Runtime { get; set; }

      
        public double? Score { get; set; }

        public virtual ICollection<GenreBioskopina> GenreMovies { get; set; } = new List<GenreBioskopina>();
    }
}

File: bioskopina_mobile_/lib/models/bioskopina.dart
import 'package:json_annotation/json_annotation.dart';
import 'genre_bioskopina.dart';

part 'bioskopina.g.dart';

@JsonSerializable()
class Bioskopina {
  final int? id;
  @JsonKey(name: 'titleEn')
  final String? titleEn;

  final String? synopsis;
  final String? director;
  @JsonKey(name: 'score')
  final double? score;
  @JsonKey(name: 'genreMovies')
  final List<GenreBioskopina>? genreMovies;
  @JsonKey(name: 'runtime')
  final int? runtime;
  final int? yearRelease;

  @JsonKey(name: 'imageUrl')
  final String? imageUrl;
  @JsonKey(name: 'trailerUrl')
  final String? trailerUrl;

  Bioskopina({
    this.id,
    this.titleEn,
    this.synopsis,
    this.director,
    this.score,
    this.genreMovies,
    this.runtime,
    this.yearRelease,
    this.imageUrl,
    this.trailerUrl,
  });

  factory Bioskopina.fromJson(Map<String, dynamic> json) =>
      _$BioskopinaFromJson(json);

  Map<String, dynamic> toJson() => _$BioskopinaToJson(this);
}
